medusa/helpers/__init__.py

```diff
@@ -422,6 +427,68 @@ def move_and_symlink_file(src_file, dest_file):
             copy_file(src_file, dest_file)
 
 
+def reflink_file(src_file, dest_file):
+    """Copy a file from source to destination with a reference link.
+
+    :param src_file: Source file
+    :type src_file: str
+    :param dest_file: Destination file
+    :type dest_file: str
+    """
+    try:
+        if reflink is None:
+            raise NotImplementedError()
+        reflink.reflink(src_file.encode('utf-8'), dest_file.encode('utf-8'))
```

Comment:
Is the encoding of `src_file` and `dest_file` really needed here? I've noticed that the reflink package supports Python 3 as well, so Unicode paths shouldn't be an issue.

Reply:
It's because otherwise reflink will encode any string as ascii, which will fail on some foreign shows.
See [this pull request](https://gitlab.com/rubdos/pyreflink/merge_requests/3) (which I just opened because I forgot about this).

Reply:
I see what you mean. In this case, can you please add a comment referencing the issue?